woocommerce/woocommerce-ios
# Simplified Login: Update prologue screen

Closes: #7899 


This PR is based on #7929 - please make sure that PR is reviewed and merged first.

### Description

This PR updates the prologue screen for the simplified login with a few changes:
- Replaced the carousel in the prologue screen with a fixed image and label
- Updated the font for the intro label
- Updated the copy for the "New to WooCommerce" link to "Learn more about WooCommerce"

Since we are still waiting for the updated image on the new prologue screen, that update will be done in a separate PR in the future.

### Testing instructions

- Reinstall the app, and notice that the same old onboarding screen is displayed on launch. The UI should be exactly as before.
- Tap Skip on the onboarding, and notice that the prologue screen should match the design in ERGtCFLyiC9CFpHsw1XIid-fi-413%3A8943 (with the Create a Store button hidden).
- Relaunch the app, the onboarding screen should not be displayed and the same prologue screen is displayed without the carousel.

With the feature flag `simplifiedLoginFlowI1` off, the onboarding and prologue screens should be the same as before this update.

### Screenshots

<img src="https://user-images.githubusercontent.com/5533851/197935876-4dd3972c-ef5a-440b-bee5-44911879b091.png" width=320 />


---
